skyportal/plot.py

```diff
@@ -756,31 +756,35 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, height=300):
     smoothed_data = pd.concat(dfs)
 
     split = data.groupby('id')
-    hover = HoverTool(
-        tooltips=[
-            ('wavelength', '$x'),
-            ('flux', '$y'),
-            ('telesecope', '@telescope'),
-            ('instrument', '@instrument'),
-            ('UTC date observed', '@date_observed'),
-            ('PI', '@pi'),
-        ]
-    )
+    # hover = HoverTool(
+    #    tooltips=[
+    #        ('wavelength', '$x'),
+    #        ('flux', '$y'),
+    #        ('telesecope', '@telescope'),
+    #        ('instrument', '@instrument'),
+    #        ('UTC date observed', '@date_observed'),
+    #        ('PI', '@pi'),
+    #    ]
+    # )
```

Comment:
This works fine on my machine.

```suggestion
     hover = HoverTool(
        tooltips=[
            ('wavelength', '$x'),
            ('flux', '$y'),
            ('telesecope', '@telescope'),
            ('instrument', '@instrument'),
            ('UTC date observed', '@date_observed'),
            ('PI', '@pi'),
        ]
     )
```

Reply:
OK.  Will submit another issue on the hover code.